tools/bcdb_mock/nodes_store.go

```diff
@@ -140,14 +140,15 @@ func (s *nodeStore) updateUptime(nodeID string, uptime int64) error {
 	return nil
 }
 
-func (s *nodeStore) StoreProof(nodeID string, dmi dmi.DMI, disks capacity.Disks) error {
+func (s *nodeStore) StoreProof(nodeID string, dmi dmi.DMI, disks capacity.Disks, hypervisor string) error {
```

Comment:
@zaibon this method signature is becoming too long, shouldn't we create (or reuse) one of the structures that already have these attributes? (same applies for other methods regarding capacity registration)

Reply:
Yep I was thinking the same.